.env.example

```
@@ -0,0 +1,14 @@
+# TrustlessWork SDK
+NEXT_PUBLIC_TRUSTLESS_WORK_API_KEY=your_api_key_here
+NEXT_PUBLIC_TRUSTLESS_WORK_ENV=development
+
+# TrustlessWork escrow roles
+NEXT_PUBLIC_PLATFORM_ADDRESS=your_platform_stellar_address
+NEXT_PUBLIC_USDC_ADDRESS=your_usdc_trustline_address
+
+# Hasura server-side access
+HASURA_GRAPHQL_URL=http://localhost:8080/v1/graphql
+HASURA_GRAPHQL_ADMIN_SECRET=myadminsecretkey
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HASURA_GRAPHQL_ADMIN_SECRET key should go before the HASURA_GRAPHQL_URL key

(UnorderedKey)